root/etc/profile.d/discovery.sh

```diff
@@ -4,25 +4,31 @@
 # in normal shells
 
 filename="/etc/default/discovery"
-while read -r line
-do
+if test -f $filename
```

Comment:
This was just reformatted and if the `$filename` is not present (extensions are not enabled), then this does not throw error during login (profile load). That was just annoying.